nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/io/JsonESDocumentWriter.java

```diff
@@ -97,6 +98,12 @@ protected void writeSystemProperties(JsonGenerator jg, DocumentModel doc) throws
         jg.writeBooleanField("ecm:isVersion", doc.isVersion());
         jg.writeBooleanField("ecm:isLatestVersion", doc.isLatestVersion());
         jg.writeBooleanField("ecm:isLatestMajorVersion", doc.isLatestMajorVersion());
+        jg.writeBooleanField("ecm:isRecord", doc.isRecord());
+        Calendar retainUntil = doc.getRetainUntil();
+        if (retainUntil != null) {
+            jg.writeStringField("ecm:retainUntil", retainUntil.toInstant().toString());
```

Comment:
`retainUntil.toInstant().toString()` -> is not an `ISO` format

Reply:
Yes it is, `Instant.toString()` is specified as being ISO format and its implementation is actually `DateTimeFormatter.ISO_INSTANT.format(this)`.